doc/Language/modules.pod6

```diff
@@ -524,6 +524,106 @@ The directories in the C<RAKULIB> path will be searched for modules when
 Raku C<need>s, C<use>s or C<require>s them. Directories that start with a
 dot are ignored and symlinks are followed.
 
+=head1 Testing modules and a distribution
+
+It is important in this section to repeat the note at the beginning of this document, namely that
+there is a difference between a B<Raku> C<distribution>, which approximates a I<module> in other languages, 
+and a B<Raku> 
+L<module declaration|https://docs.raku.org/language/syntax#index-entry-declarator_unit-declarator_module-declarator_package-Package,_Module,_Class,_Role,_and_Grammar_declaration>.
+The reason for this is that a single file may contain a number of C<class>, C<module>, C<role> etc declarations, or
+these declarations may be spread between different files. In addition, when a C<distribution> is published (see 
+L<Distributing modules>) it may be necessary to provide access to other resources, such as callable utilities.
+
+B<Raku> also allows for a single distribution to provide multiple C<compunit>s that can be I<use'd> (or I<require'd> etc).
+The information
+about a distribution is contained in the C<META6.json> file in the distribution's root directory. See below
+for more about C<META6.json>. Each entity that the B<distribution> allows to be I<use'd> (or I<require'd> etc), 
+also called a C<compunit>, is 
+placed in the C<provides> section of the C<META6.json> (as specified below). 
+
+It should also be noted that when 
+writing the C<depends> section of the C<META6.json> file, it is the name of the B<distribution> that is listed,
```

Comment:
Well... you know all the trouble about Pod::To::HTML? It was because what you include in depends, or at least how it's interpreted, is the name of the module, not the distribution; actually, it looks up the "provides" section of every distribution.

Reply:
It was trying to resolve those problems that led me to want to change the documentation. 
However, I think that may be it would be better to commit these changes to the module document, then add more information with another PR.

Reply:
You need to change "distribution" to "module" then; depends list modules, not distributions.